public/API/API_GetTicketData.php

```diff
@@ -209,36 +209,70 @@
             $prevID = $ticket['AchievementID'];
         }
     }
-    $ticketData['URL'] = $baseUrl . "?u=$assignedToUser";
+    $ticketData['URL'] = route('developer.tickets', $assignedToUser);
 
     return response()->json($ticketData);
 }
 
+$getTicketsInfo = function (Builder $tickets, int $offset, int $count): array {
+    $result = [];
+
+    /** @var Ticket $ticket */
+    foreach ($tickets->orderBy('ReportedAt', 'DESC')->offset($offset)->take($count)->get() as $ticket) {
+        $ticket->loadMissing(['achievement', 'reporter', 'resolver']);
+        $ticket->achievement->loadMissing('game');
+        $ticket->achievement->game->loadMissing('system');
```

Comment:
I'm particularly concerned about this as it's inducing an N+1 query problem. We should perform these loads outside the loop if possible.

```php
$tickets = $tickets->with([
    'achievement.game.system',
    'reporter',
    'resolver',
])
    ->orderBy('ReportedAt', 'DESC')
    ->offset($offset)
    ->take($count)
    ->get();
```

Reply:
done